cmake/modules/FindIconv.cmake

```diff
@@ -0,0 +1,35 @@
+# FindIconv
+# ----------
+# Finds the Iconv library
+#
+# This will define the following variables:
+#
+#  ICONV_FOUND - system has Iconv
+#  ICONV_LIBRARIES - Link these to use Iconv
+
+include(FindPackageHandleStandardArgs)
+
+check_library_exists("" iconv_open "iconv.h" HAVE_ICONV_IN_LIBC)
```

Comment:
this doesn't work like you think. LOCATION is intended to be a directory, not a header file.
see https://github.com/Kitware/CMake/blob/master/Modules/CheckLibraryExists.cmake#L61